runtime/doc/lua.txt

```diff
@@ -4308,4 +4308,30 @@ vim.text.hexencode({str})                               *vim.text.hexencode()*
     Return: ~
         (`string`) Hex encoded string
 
+
+==============================================================================
+Lua module: tohtml                                                    *tohtml*
+
+
+:TOhtml {file}                                                       *:TOhtml*
+Generates HTML and writes to a file and opens it in a new split window. If the
+file is not given then it uses a temporary file.
```

Comment:
```suggestion
Converts the buffer shown in the current window to HTML, opens the generated HTML in a new split window, and saves its contents to {file}. If {file} is not given, a temporary file is used.
```
Does it really create a "temporary file"? Or does it just skips setting the buffer name and writing to a file?

Reply:
It does create a temporary file (using `writefile()`).

Reply:
Where (current dir, tempdir) and with what name? (Might be good to note here.)

Reply:
Okay, I added note to doc about how `tempname()` is used to create the file.